.NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchTimeExtractorConfiguration.cs

```diff
@@ -0,0 +1,184 @@
+        public static readonly Regex PmRegex =
+            new Regex(
+                DateTimeDefinitions.PmRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        // handle "... dans la matinee (in the morning)"
+        public static readonly Regex AmRegex = 
+            new Regex(
+                DateTimeDefinitions.AmRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        // handle "half past ..." "a quarter to ..."
+        // rename 'min' group to 'deltamin'
+        public static readonly Regex LessThanOneHour =
+            new Regex(
+                DateTimeDefinitions.LessThanOneHour,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex EngTimeRegex =
+            new Regex(
+                DateTimeDefinitions.EngTimeRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        // TODO - will have change below
+        // handle "six heures et demie" (six thirty), "six heures et vingt-et-un" (six twenty one) 
+        public static readonly Regex TimePrefix =
+            new Regex(
+                DateTimeDefinitions.TimePrefix,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex TimeSuffix =
+            new Regex(
+                DateTimeDefinitions.TimeSuffix, 
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex BasicTime =
+            new Regex(
+                DateTimeDefinitions.BasicTime,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        // handle special time such as 'at midnight', 'midnight', 'midday'
+        // midnight - le minuit, la zero heure
+        // midday - midi 
+
+        public static readonly Regex MidnightRegex =
+            new Regex(
+                DateTimeDefinitions.MidnightRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MidmorningRegex =
+            new Regex(
+                DateTimeDefinitions.MidmorningRegex, 
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MidafternoonRegex =
+            new Regex(
+                DateTimeDefinitions.MidafternoonRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MiddayRegex =
+            new Regex(
+                DateTimeDefinitions.MiddayRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MidTimeRegex =
+            new Regex(
+                DateTimeDefinitions.MidTimeRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        // part 3: regex for time
+        // --------------------------------------
+        // handle "at four" "at 3"
```

Comment:
I see you've translated some of the example comments to French. Can you do the same for the rest of them?